File: LeetCode/Python/1295_Find_Numbers_with_Even_Number_of_Digits.py
# 1295. Find Numbers with Even Number of Digits
# https://leetcode.com/problems/find-numbers-with-even-number-of-digits/


class Solution:
    def findNumbers(self, nums: list[int]) -> int:
        count = 0

        for num in nums:
            if len(str(num)) % 2 == 0:
                count += 1

        return count

File: LeetCode/JavaScript/1295-Find-Numbers-with-Even-Number-of-Digits.js
// 1295. Find Numbers with Even Number of Digits
// https://leetcode.com/problems/find-numbers-with-even-number-of-digits/

/**
 * @param {number[]} nums
 * @return {number}
 */

var findNumbers = function (nums) {
  let count = 0;

  for (let num of nums) {
    if (num.toString().length % 2 === 0) count++;
  }

  return count;
};
